### ISSUE TITLE
SpecialGlobalVars add EnforcedStyle english_when_available

### ISSUE THREAD

**Issue by grosser** [bbatsov/rubocop#5891]
problem:
 - using `$PID` is not safe without `require 'English'`
 - using `$:` is ugly but `$LOAD_PATH` is defined

solution:
Add a new style that uses `$LOAD_PATH` but not `$PID` ?

**Comment by bbatsov** [comment]
The problem is you can't really know if `English` is loaded using just static analysis. I see your point, I just wonder whether it makes sense to make thing more complex if people can just added a single require high enough in their apps/libs.

**Comment by bbatsov** [comment]
Yeah, I get this - I just wonder whether it's worth adding such style. I'd like to hear more opinions on the subject.

**Comment by splattael** [comment]
:wave: @bbatsov 

At GitLab we've discussed the very same issue to allow all [`NON_ENGLISH_VARS`](https://github.com/rubocop/rubocop/blob/34261372a4f51fce3e1b8ceb0d81784df0d8b1d7/lib/rubocop/cop/style/special_global_vars.rb#L102) (e.g. `$LOADED_FEATURES` etc.) but enforce Perl names (e.g. `$?`) otherwise.

Would you accept a PR which adds a new `EncforcedStyle: use_builtin` which behaves like `EncforcedStyle: use_perl_names` except it allows [`NON_ENGLISH_VARS`](https://github.com/rubocop/rubocop/blob/34261372a4f51fce3e1b8ceb0d81784df0d8b1d7/lib/rubocop/cop/style/special_global_vars.rb#L102)? :thinking: 

Alternatively, we'd go with `EncforcedStyle: use_english_names` + `require "English"` :muscle: 

/cc @grosser
